diff --git a/owamp/owamp/selinux/owamp-server.te b/owamp/owamp/selinux/owamp-server.te
index d1342a2..508f1ad 100644
--- a/owamp/owamp/selinux/owamp-server.te
+++ b/owamp/owamp/selinux/owamp-server.te
@@ -71,6 +71,7 @@ files_tmp_filetrans(owampd_t, owampd_tmp_t, file)
 allow owampd_t self:capability net_bind_service;
 corenet_tcp_bind_generic_node(owampd_t)
 allow owampd_t owamp_port_t:tcp_socket { name_bind };
+allow owampd_t hi_reserved_port_t:tcp_socket { name_bind name_connect };
 allow owampd_t self:tcp_socket { listen accept };
 corenet_udp_bind_generic_node(owampd_t)
 allow owampd_t owamp_test_port_t:udp_socket { name_bind };
diff --git a/owamp/owamp/selinux/twamp-server.te b/owamp/owamp/selinux/twamp-server.te
index 99f4841..0695636 100644
--- a/owamp/owamp/selinux/twamp-server.te
+++ b/owamp/owamp/selinux/twamp-server.te
@@ -63,6 +63,7 @@ manage_files_pattern(twampd_t, twampd_var_lib_t, twampd_var_lib_t)
 allow twampd_t self:capability net_bind_service;
 corenet_tcp_bind_generic_node(twampd_t)
 allow twampd_t twamp_port_t:tcp_socket { name_bind };
+allow twamp_t hi_reserved_port_t:tcp_socket { name_bind name_connect };
 allow twampd_t self:tcp_socket { listen accept };
 corenet_udp_bind_generic_node(twampd_t)
 allow twampd_t twamp_test_port_t:udp_socket { name_bind };
